src/static/js/wishlist.js

```diff
@@ -91,3 +91,45 @@ setTimeout(() => {
   document.querySelectorAll('.upForm').forEach(element => { listen(element, 'up') })
   document.querySelectorAll('.downForm').forEach(element => { listen(element, 'down') })
 }, 0)
+
+function priceToFloat(price) {
+  let cleanPrice = price
+      .replace(/[^\d.,]/g, '')  // Remove everything except digits, periods, and commas
+      .replace(/,/g, '');       // Remove commas to standardize the number
+  // If there's more than one decimal separator (.), treat the last one as the decimal point
+  if ((cleanPrice.match(/\./g) || []).length > 1) {
+      cleanPrice = cleanPrice.replace(/\.(?=.*\.)/, '');  // Remove the first period if there are multiple
+  }
+  return parseFloat(cleanPrice);
+}
+
+document.addEventListener('DOMContentLoaded', function(){
+  let sortOrderPrice = 'asc';
+  const priceArrow = document.getElementById('price-arrow');
+  const sortByPrice = document.getElementById('sort-price');
+
+  function updateArrow(column, order) {
+    if (column === 'price') {
+      priceArrow.className = order === 'asc' ? 'fas fa-arrow-up' : 'fas fa-arrow-down';
+    }
+  }
+
+  sortByPrice.addEventListener('click', function(e){
+    e.preventDefault();
+
+    const table = document.getElementById('wishlist-table');
+    const tbody = table.querySelector('tbody');
+    const rows = Array.from(tbody.querySelectorAll('tr'));
+    sortOrderPrice = sortOrderPrice === 'asc' ? 'desc' : 'asc';
+
+    const sortedRows = rows.sort((a,b) => {
+      priceA = priceToFloat(a.cells[4].textContent)
```

Comment:
Also make sure to put `const` before your variables here.

You could also avoid duplicating it with `const [priceA, priceB] = [a, b].map(x => x.querySelector('.price').textContent)`, but that adds complexity that I'm not sure is worth it.

Reply:
Resolved with https://github.com/Wingysam/Christmas-Community/pull/140/commits/67b2a5f2e96f275b3971c5a25d4c29e5b0b7a802